code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -296,3 +296,10 @@
 /obj/item/weapon/storage/box/syndie_kit/mimery/PopulateContents()
 	new /obj/item/weapon/spellbook/oneuse/mimery_blockade(src)
 	new /obj/item/weapon/spellbook/oneuse/mimery_guns(src)
+
+/obj/item/weapon/storage/box/syndie_kit/holoparasite
```

Comment:
There should be a special syndibox sprite somewhere. The box can be folded up if you're trying to be sneaky (with a guardian...?) anyway, but I'm a nerd about aesthetics. Use it, if you like.